cli/commands/project_sync.go

```diff
@@ -97,56 +97,60 @@ func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 	}
 
 	// Determine the working directory for CWD-sensitive operations.
-	if workingDir, err := cmd.DeriveLocalSyncPath(cmd.Config, ctx.String("dir")); err == nil {
-		// Determine the volume name to be used across all operating systems.
-		// For cross-compatibility the way this volume is set up will vary.
-		volumeName := cmd.GetVolumeName(ctx, cmd.Config, workingDir)
-
-		switch platform := runtime.GOOS; platform {
-		case "linux":
-			cmd.out.Verbose.Printf("Setting up local volume: %s", volumeName)
-			cmd.SetupBindVolume(volumeName, workingDir)
-		default:
-			cmd.out.Verbose.Printf("Starting sync with volume: %s", volumeName)
-			cmd.StartUnisonSync(ctx, volumeName, cmd.Config, workingDir)
-		}
-	} else {
-		cmd.out.Error.Fatal(err)
+	var workingDir, err = cmd.DeriveLocalSyncPath(cmd.Config, ctx.String("dir"))
```

Comment:
Why does this work without the `:=`? When I've tried things like this before, I'd have err yelling at me.

Reply:
:= is just a shorthand for not needing to `var` it. The var and = I believe is redefining it.